pkg/exporter/sumologicexporter/transform_metadata.go

```diff
@@ -0,0 +1,37 @@
+package sumologicexporter
```

Comment:
missing header

Reply:
That's correct. I want to ask here:
1) Is the part `// Copyright 2021, OpenTelemetry Authors` correct? Are we _OpenTelemetry Authors_ here in this repo?
2) Do we need all those headers in the first place? I haven't seen headers in other repositories of ours.

Reply:
From the discussion in the sync:
1. We should be using `Sumo Logic, Inc.` as the author for code originally written in Sumo Logic and specific to Sumo Logic projects (as opposed to upstream projects like [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector), [OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/) etc.)
2. Yes we need the headers, we should probably stick to what the License says in the appendix: https://www.apache.org/licenses/LICENSE-2.0.html#apply

Reply:
Added header.